src/pages/home/report/ReportActionItem.js

```diff
@@ -190,7 +190,7 @@ class ReportActionItem extends Component {
                                 <OfflineWithFeedback
                                     onClose={() => {
                                         if (this.props.action.pendingAction === CONST.RED_BRICK_ROAD_PENDING_ACTION.ADD) {
-                                            ReportActions.deleteOptimisticReportAction(this.props.report.reportID, this.props.action.sequenceNumber);
+                                            ReportActions.deleteOptimisticReportAction(this.props.report.reportID, this.props.action.clientID);
```

Comment:
Why did we change this to `clientID`?  `ReportActions.deleteOptimisticReportAction` expects `sequenceNumber`